src/char/reader.cr

```diff
@@ -14,11 +14,31 @@ struct Char
   struct Reader
     include Enumerable(Char)
 
+    # Returns the reader's String.
     getter string
+
+    # Returns the current character.
+    # ```
+    # reader = Char::Reader.new("ab")
+    # reader.current_char #=> 'a'
+    # reader.next_char
+    # reader.current_char #=> 'b'
+    # ```
     getter current_char
+
+    #:nodoc:
     getter current_char_width
+
+    # Returns the position of the current character.
+    # ```
+    # reader = Char::Reader.new("ab")
+    # reader.current_char #=> 0
```

Comment:
`crystal tool format` formats `#=>` as `# =>`.

Please run `crystal tool format` to ensure the changes match the standard defined. Thank you.